packages/experimental/Button/src/Button.styling.ts

```diff
@@ -0,0 +1,82 @@
+import { buttonName, ButtonTokens, ButtonSlotProps, ButtonProps } from './Button.types';
+import { ITheme, UseStylingOptions, buildProps } from '@fluentui-react-native/experimental-framework';
+import { borderStyles, fontStyles } from '@fluentui-react-native/tokens';
+
+export const stylingSettings: UseStylingOptions<ButtonProps, ButtonSlotProps, ButtonTokens> = {
+  tokens: [
+    (t: ITheme) => ({
+      backgroundColor: t.colors.buttonBackground,
+      color: t.colors.buttonText,
+      borderColor: t.colors.buttonBorder,
+      borderWidth: 1,
+      borderRadius: 2,
+      disabled: {
+        backgroundColor: t.colors.buttonBackgroundDisabled,
+        color: t.colors.buttonTextDisabled,
+        borderColor: t.colors.buttonBorderDisabled
+      },
+      hovered: {
+        backgroundColor: t.colors.buttonBackgroundHovered,
+        color: t.colors.buttonTextHovered,
+        borderColor: t.colors.buttonBorderHovered as string
+      },
+      pressed: {
+        backgroundColor: t.colors.buttonBackgroundPressed,
+        color: t.colors.buttonTextPressed,
+        borderColor: t.colors.buttonBorderPressed as string
+      },
+      focused: {
+        borderColor: t.colors.buttonBorderFocused,
+        color: t.colors.buttonTextHovered
+      }
+    }),
+    buttonName
+  ],
+  states: ['hovered', 'focused', 'pressed', 'disabled'],
```

Comment:
This replaces __overrides, right? How does this change for nested states (focused + pressed or whatever)

Reply:
I saw your response to Krystal elsewhere, this is still recursive